api/src/main/kotlin/nl/avisi/anowalke/services/VertaalService.kt

```diff
@@ -0,0 +1,83 @@
+package nl.avisi.anowalke.services
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.dto.Translations
+import nl.avisi.anowalke.util.configureObjectMapper
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.ResponseEntity
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+import org.springframework.stereotype.Component
+import org.springframework.web.client.HttpStatusCodeException
+import org.springframework.web.client.RestTemplate
+
+@Component
+class VertaalService {
+
+    val dummylist = listOf("subscription","key","region","endpoint","url","client","inputlist","request","body","media","type","gson","response","list","of","my","class","object","translations","it","post","json_text","parser","json","prettify","translator")
+
+    private val subscriptionKey = "f2a9a9f29d7d4e878669cdfed04f99ce"
```

Comment:
Dergelijke keys zijn niet handig om op te nemen in je repository. Beter is om dit te doen met ConfigurationProperties (https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/context/properties/ConfigurationProperties.html) Je kan dan in de application.yaml waarden aanmaken welke gemapt worden op een data class, welke je vervolgens weer hier kunt gebruiken. Om het af te maken kan je dan in die application.yaml dummy waarden neer zetten en deze waarden overschrijven middels environment variables(https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html) Dit commit je niet mee naar je repo. In dit geval al helemaal handig omdat ik zie dat je repo public is en dus iedereen deze key kan achterhalen en gebruiken.

Reply:
In die properties kan je dan ook onderstaande waarde configureren indien gewenst